docs/_docs/2_4_configurations.md

```diff
@@ -396,56 +396,90 @@ Property: `hoodie.compaction.daybased.target` <br/>
 Property: `hoodie.compaction.payload.class` <br/>
 <span style="color:grey">This needs to be same as class used during insert/upserts. Just like writing, compaction also uses the record payload class to merge records in the log against each other, merge again with the base file and produce the final record to be written after compaction.</span>
 
-
 ### Metrics configs
+
 Enables reporting on Hudi metrics.
 [withMetricsConfig](#withMetricsConfig) (HoodieMetricsConfig) <br/>
-<span style="color:grey">Hudi publishes metrics on every commit, clean, rollback etc.The following sections list the supported reporters.</span>
+<span style="color:grey">Hudi publishes metrics on every commit, clean, rollback etc. The following sections list the supported reporters.</span>
 
 #### GRAPHITE
 
-##### on(metricsOn = true) {#on}
+##### withMetricsOn(metricsOn = false) {#withMetricsOn}
 `hoodie.metrics.on` <br/>
-<span style="color:grey">Turn sending metrics on/off. on by default.</span>
+<span style="color:grey">Turn on/off metrics reporting. off by default.</span>
 
 ##### withReporterType(reporterType = GRAPHITE) {#withReporterType}
 Property: `hoodie.metrics.reporter.type` <br/>
 <span style="color:grey">Type of metrics reporter.</span>
 
-##### toGraphiteHost(host = localhost) {#toGraphiteHost}
+##### withGraphiteHost(host = localhost) {#withGraphiteHost}
 Property: `hoodie.metrics.graphite.host` <br/>
 <span style="color:grey">Graphite host to connect to</span>
 
-##### onGraphitePort(port = 4756) {#onGraphitePort}
+##### withGraphitePort(port = 4756) {#withGraphitePort}
 Property: `hoodie.metrics.graphite.port` <br/>
 <span style="color:grey">Graphite port to connect to</span>
 
-##### usePrefix(prefix = "") {#usePrefix}
+##### withGraphitePrefix(prefix = "") {#withGraphitePrefix}
 Property: `hoodie.metrics.graphite.metric.prefix` <br/>
 <span style="color:grey">Standard prefix applied to all metrics. This helps to add datacenter, environment information for e.g</span>
 
 #### JMX
 
-##### on(metricsOn = true) {#on}
+##### withMetricsOn(metricsOn = false) {#withMetricsOn}
 `hoodie.metrics.on` <br/>
-<span style="color:grey">Turn sending metrics on/off. on by default.</span>
+<span style="color:grey">Turn on/off metrics reporting. off by default.</span>
 
 ##### withReporterType(reporterType = JMX) {#withReporterType}
 Property: `hoodie.metrics.reporter.type` <br/>
 <span style="color:grey">Type of metrics reporter.</span>
 
-##### toJmxHost(host = localhost) {#toJmxHost}
+##### withJmxHost(host = localhost) {#withJmxHost}
 Property: `hoodie.metrics.jmx.host` <br/>
 <span style="color:grey">Jmx host to connect to</span>
 
-##### onJmxPort(port = 1000-5000) {#onJmxPort}
+##### withJmxPort(port = 1000-5000) {#withJmxPort}
```

Comment:
Trying to make `with` the same prefix for this builder pattern. IMO having different prepositions is just confusing.